generate_endf.py

```diff
@@ -32,7 +33,7 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
 )
 
 parser.add_argument('-d', '--destination', type=Path,
-                    default=Path('endf-b7.1-hdf5'),
+                    default=None,
```

Comment:
`None` is the default anyway, so you can remove this

Reply:
Thanks, I didn't realize that, I've now removed that line as suggested